apps/learninglocker/templates/services/xapi/dc.yml.j2

```diff
@@ -35,6 +35,10 @@ spec:
                         - "learninglocker-xapi-{{ deployment_stamp }}"
                 topologyKey: kubernetes.io/hostname
       containers:
+{% if learninglocker_xapi_image_pull_secret_name is defined and learninglocker_xapi_image_pull_secret_name is not none %}
```

Comment:
:warning: This `imagePullsecrets` block has been mixed with the `containers` block.

Reply:
good catch. I corrected!